lib/debezium/decimal.go

```diff
@@ -0,0 +1,82 @@
+package debezium
+
+import (
+	"math/big"
+
+	"github.com/artie-labs/transfer/lib/typing/decimal"
+)
+
+func EncodeDecimal(value string, scale int) []byte {
```

Comment:
Copied this verbatim from https://github.com/artie-labs/reader/blob/529d2f3181db4586c97be641ad3f520a0ed9ea27/lib/debezium/converters/decimal.go#L11